lib/binary/index.js

```diff
@@ -0,0 +1,5 @@
+export default function binary(cb) {
+  return function(a, b) {
+    return cb(a, b)
+  }
+}
```

Comment:
curry here?

Reply:
`nAry` doesn't curry function, so I think `unary` and `binary` shouldn't

Reply:
oke